### DISCUSSION TITLE
Can't Boost to desktop jetson nano with Image jetbot-043_nano-4gb-jp45.zip

### DISCUSSION THREAD

**Discussion by ThaiVuz** [NVIDIA-AI-IOT/jetbot#541] [Q&A]
For my Jetson Nano (4GB) Dev Kit - B01, I have been using jetbot-043_nano-4gb-jp45.zip. I can’t start up Jetson nano to the desktop. When I power on CUI will be displayed instead of the expected GUI.
Then enter the ‘unminimize’ command and ‘y’ according to the message suggestion, but I receive the massage as picture below.
Can you help me how to access the desktop jetson nano??? Please help me !!! I search a lot of forum but I can’t find solution !!!
![1a7b6facce960ec85787](https://user-images.githubusercontent.com/93064940/173167838-3f514664-aa1c-44f3-868d-358bfad68a66.jpg)

**Comment by AliFarzi** [comment]
You can use code below for enabling the gui. 
`sudo systemctl set-default graphical.target
`

This is a code for disabling the gui
`sudo systemctl set-default multi-user
`


Alternatively, run `/jetbot/scripts/re_enable_gui.sh`

Restart the device after executing.
